cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -349,7 +349,7 @@ public void testGenerateStatefulSetWithEmptyStorageSelector() {
 
     @Test
     public void testGenerateStatefulSetWithSetSizeLimit() {
-        String sizeLimit = "1Gi";
+        String sizeLimit = "1";
```

Comment:
This doesn't sound right to me. Why is this needed nd why is this ok to change it?